examples/with_winit/src/lib.rs

```diff
@@ -658,7 +658,8 @@ fn android_main(app: AndroidApp) {
 
     let event_loop = EventLoopBuilder::with_user_event()
         .with_android_app(app)
-        .build();
+        .build()
+        .expect("Required to continue");
```

Comment:
It looks like it only errors in the case that you call build() multiple times, which in 0.28 was a panic.
So adding an expect there would just be continuing that panic tradition.

So this seems fine to me.

Reply:
Thanks, that makes sense